src/Microsoft.ML.StandardLearners/Standard/LinearModelParameters.cs

```diff
@@ -455,9 +453,19 @@ private LinearBinaryModelParameters(IHostEnvironment env, ModelLoadContext ctx)
 
             // *** Binary format ***
             // (Base class)
-            // LinearModelStatistics: model statistics (optional, in a separate stream)
-
-            ctx.LoadModelOrNull<LinearModelStatistics, SignatureLoadModel>(Host, out _stats, ModelStatsSubModelFilename);
+            // LinearModelParameterStatistics: model statistics (optional, in a separate stream)
+            try
+            {
+                LinearModelParameterStatistics stats;
+                ctx.LoadModelOrNull<LinearModelParameterStatistics, SignatureLoadModel>(Host, out stats, ModelStatsSubModelFilename);
```

Comment:
>LoadModelOrNull [](start = 20, length = 15)

Does this throw on failure, or just return null?

Reply:
throw if the signature supplied is not correct. 

---
In reply to: [264870883](https://github.com/dotnet/machinelearning/pull/2048#discussion_r264870883) [](ancestors = 264870883)